public/stylesheets/style.scss

```diff
@@ -1,8 +1,135 @@
-body {
-  padding: 50px;
+$sand-top: #D7D3C8;
```

Comment:
since we're using sass, we should try to consolidate all these styles into a readable hierarchy, as opposed to just having everything laid out like normal CSS.